hapi/src/services/eosio.service.js

```diff
@@ -24,6 +24,8 @@ const getProducers = async () => {
     producers.push(...rows)
   }
 
+  producers = producers.filter(producer => !!producer.is_active)
```

Comment:
will inactive block producers that were already added to the datbase be removed?

what is a BP was active and then becomes inactive?

Reply:
every day we are doing a complete update of the producer, so if a producer was active and now it's no longer active will be removed if it will be active at any time it will be included on our database